magento/magento2
# Fix #13401 - Multi-Store: "Store View" sort order values are not reflected in front-end store-switcher

### Description (*)

This fixes #13401
However I only modified sort order on FE, since on admin grid it seems to be sorted alphabetically, and not sure if that should be changed. If yes, I can adjust PR.

### Related Pull Requests


### Fixed Issues (if relevant)

1. magento/magento2#13401: Multi-Store: "Store View" sort order values are not reflected in the admin or front-end store-switcher

### Manual testing scenarios (*)

1. Create one website, with three Store Groups Each with 1 store
2. Within the Store view 1, set priority of 0, Store View 2, priority of 10, Store View 3, priority of 20
3. Review store switcher on front-end, sort order should impact order of stores visible

### Questions or comments


### Contribution checklist (*)
